docs/features/models/kbqa.rst

```diff
@@ -4,54 +4,94 @@ Knowledge Base Question Answering (KBQA)
 Description
 -----------
 
-The Knowledge Base Question Answering model uses Wikidata to answer question. To find the answer the following
+There are three models for KBQA in DeepPavlov library: model for answering complex questions in English, model for answering simple questions in Russian and model for answering simple questions in Russian using syntactic parser.
```

Comment:
It isn't clear what these models are. Maybe it's better to write a list with links to these models:
There are three models for KBQA in DeepPavlov library:
 - :config:`kbqa_cq <kbqa/kbqa_cq.json>` - model for....
Construction :config:`kbqa_cq <kbqa/kbqa_cq.json>` creates link in documentation to json file on github